.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/AssistantTests.cs

```diff
@@ -22,6 +22,7 @@ namespace Azure.AI.OpenAI.Tests;
 
 public class AssistantTests(bool isAsync) : AoaiTestBase<AssistantClient>(isAsync)
 {
+#if !AZURE_OPENAI_GA
```

Comment:
Not for this PR but for future reference:

Not sure if this part of the test framework works in my repo, but in the Azure .NET repo it is possible to set an attribute to a test class or a test function to say which versions it supports. That is because we run each test against all versions that are supposedly valid.

For tests that only work in the preview, you can add:
`[ServiceVersion(Min = AzureOpenAIClientOptions.ServiceVersion.V2024_08_01_Preview)]`

Reply:
yeah, that'll be a nice thing for us to investigate and use!